plugins/ipc/management/commands/load_ipc_status.py

```diff
@@ -84,17 +85,23 @@ def handle(self, *args, **options):
         updated = timezone.now()
         updated_by = User.objects.filter(username='ohc').first()
 
-        inpatients = BedStatus.objects.filter(bed_status='Occupied').values_list(
-            'local_patient_identifier', flat=True)
-
+        inpatient_ids = set(BedStatus.objects.filter(bed_status='Occupied').values_list(
+            'patient_id', flat=True
+        ))
         upstream_result = api.execute_hospital_query(QUERY)
-
         self.stdout.write("Query complete")
 
+        mrn_to_ipc_patients = find_patients_from_mrns(
+            i['Patient_Number'] for i in upstream_result
+        )
+
         for row in upstream_result:
-            if row['Patient_Number'] in inpatients:
-                patient = Patient.objects.get(demographics__hospital_number=row['Patient_Number'])
+            patient = mrn_to_ipc_patients.get(row['Patient_Number'])
+
+            if not patient:
+                continue
 
+            if patient.id in inpatient_ids:
```

Comment:
We might have done this for some testing purpose, but this (and L88-90) above shouldn't have crept into a release branch - suggest we take it out now in case we forget it's here

Reply:
Are we talking about either

a) Removing IPC creation
b) Changing it so that regardless of bed status IPC statuses are created.